src/main/java/com/stripe/BranchCoverageUtil.java

```diff
@@ -0,0 +1,67 @@
+package com.stripe;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.io.File;
+
+public class BranchCoverageUtil {
+  public static void writeDefault() {
```

Comment:
You could call this `init` or sth idk im just writing this comment so that they know that we know how to use Github.

Reply:
"I ain't the sharpest tool in the shed" - I think you can relate to that. I will respectfully ignore that suggestion - I know how to use Github master TA san.